lib/cdo/aws/s3.rb

```diff
@@ -371,6 +371,11 @@ def upload_file(filename, options={})
           return upload_log(File.basename(filename), file, options)
         end
       end
+
+      module LogBucketNames
+        # Bucket used for internal audit logs.
+        AUDIT_LOGS_BUCKET = 'cdo-audit-logs'.freeze
+      end
```

Comment:
This is new, can you provide some context?

Reply:
It seems we should use `CDO` instead of a constant:
https://github.com/code-dot-org/code-dot-org/blob/staging/config.yml.erb#L256-L270

Reply:
Yep moved it to config.yml